cirq/google/common_serialized_gates_test.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from cirq import ops
```

Comment:
for tests our pattern is to use as close to what we'd like the user to use.  So
```
import cirq
import cirq.google as cg
```
and then use these.

Reply:
Done.